### DISCUSSION TITLE
A bug browser/UI doesn't stop counting down after each time switching model/checkpoint? What's this bug usually refered?

### DISCUSSION THREAD

**Discussion by nnqq2603** [vladmandic/sdnext#1401] [Q&A]
![image](https://github.com/vladmandic/automatic/assets/116576065/9e3b3b96-0603-40e9-a14f-9f83b813a557)

So I'm in **Version: 1d0a18ef Sun Jun 11 09:00:38 2023 -0400** (just update last Sunday).

I did close and re-open browser, server command console several times but this bug is still remaining. When I fresh start UI, run browser and open `127.0.0.1:7860` **the count down keep running regardless of the real status of model loading**, at first I thought my HDD has problem and it can't load checkpoint/model but then it counts down for half hour and even more. So I turn off everything, re-run again, same issue, but after a few times, I try generate img while the counting down still run, and it works, like the UI is responsive and functional, I still can do all txt2img, img2img... but can't click and load other model using the quick-scrolldown-menu. I still can swith model/checkpoint by using Show/Hide Extra Networks button and click on cards gallery though. _(This issue keep happening with different checkpoints, I tried change the start-up load model in config file and issue is still remaning)_

English isn't my native language, I'm not sure how to call this bug or the more accurate term to refer this bug (so I can't search for it well).

**Comment by vladmandic** [comment]
first, read through #1113
second, posting 10 pages of log dump doesnt help - who is going to go over that log? especially since this is a client-side issue and server log is not expected to show anything.

**Reply by nnqq2603** [reply]
Thanks, I can't search because the previous question use another term to describe bug in their title. I wish I could know what useful or whatnot, I literally have no idea that why I ask the question. Sorry because it's duplicated.
